proto/api.proto

```diff
@@ -408,16 +408,16 @@ message AdminSplitResponse {
 }
 
 // An AdminMergeRequest is arguments to the AdminMerge() method. A
-// merge is always performed by calling AdminMerge on the range
-// that is subsuming the passed in subsumed_range. The ranges must
-// be consecutive in the key space, such that the end_key of the
-// subsuming range must match the start_key of the range being subsumed.
-// After the merge operation, the subsumed_range will no longer exist and
-// the subsuming range will now encompass all keys from its original
-// start_key to the end_key of the subsumed_range.
+// merge is performed by calling AdminMerge on the left-hand range of
+// two consecutive ranges (i.e. the range which contains keys which
+// sort first). This range will be the subsuming range and the right
+// hand range will be subsumed. After the merge operation, the
+// subsumed_range will no longer exist and the subsuming range will
+// now encompass all keys from its original start_key to the end_key
+// of the subsumed_range. If AdminMerge is called on the final range
+// in the key space, it is a noop.
```

Comment:
Actually, this won't be a requirement at this level. Calling AdminMerge will start an op on the server owning the subsuming range which will start by moving the replicas to the range and then will execute the merge.